### PR TITLE
Update Censys Searcher

### PR DESCRIPTION
**Pull request by thehappydinoa**
- Updated queries
- Added support for `gaTrackID`

### CODE REVIEW

**Comment by thehappydinoa** [comment]
Hey @ninoseki, 

Would you be able to take a look at this PR?
I would love to be able to use the updated version. Hope all is well!

File: src/searcher/censys.ts [hunk 1]
```diff
@@ -18,22 +18,29 @@ export class Censys implements Searcher {
   public searchByASN(query: string): string {
     const asn = extractASNumber(query);
     return buildURL(this.baseURL, "/search", {
-      q: `autonomous_system.asn:${asn}`,
+      q: `autonomous_system.asn: ${asn}`,
```
Comment (ninoseki) [inline src/searcher/censys.ts:21]: Does it really need a space after the search filter?

File: src/searcher/censys.ts [hunk 2]
```diff
@@ -18,22 +18,29 @@ export class Censys implements Searcher {
...
       resource: "certificates",
     });
   }
+
+  public searchByGATrackID(query: string): string {
+    return buildURL(this.baseURL, "/search", {
+      q: `services.http.response.body: "${query}"`,
+      resource: "hosts",
+    });
+  }
```
Comment (ninoseki) [inline src/searcher/censys.ts:45]: I understand your intent. But I think it's not good. It's misleading for users.
I think many users think that Censys has a special feature to search hosts by GA track ID, but it doesn't. 
Also, if I allow this, I should add the same method for Shodan, Onyphe, etc. It doesn't make much sense to me.

So could you remove this function and update docs, please?

**Review by ninoseki — CHANGES_REQUESTED** [review]

File: src/searcher/censys.ts (hunk 2 above)
Comment (thehappydinoa) [reply src/searcher/censys.ts:45]: Totally fair, I will update it.

File: src/searcher/censys.ts (hunk 1 above)
Comment (thehappydinoa) [reply src/searcher/censys.ts:21]: Not necessarily, its more stylistic.

**Comment by ninoseki** [comment]
Let me close this PR & please re-open it if you want to work on this PR again.

**Pull request closed without merging**